packages/react-examples/.eslintrc.json

```diff
@@ -5,6 +5,7 @@
     "import/no-webpack-loader-syntax": "off",
     "@typescript-eslint/no-explicit-any": "off",
     "no-alert": "off",
-    "no-restricted-globals": "off"
+    "no-restricted-globals": "off",
+    "@typescript-eslint/no-deprecated": "warn"
```

Comment:
discussed offline: needs to be reverted as this is published package and we dont wanna allow further usage of deprecated apis for public api. will be handled in https://github.com/microsoft/fluentui/pull/34157